src/tezos_interop/tezos_interop.ml

```diff
@@ -172,7 +111,9 @@ module Consensus = struct
     transactions : transaction list;
   }
   let parse_transaction transaction =
-    match (transaction.Listen_transactions.entrypoint, transaction.value) with
+    let Tezos_bridge.Listen_transaction.{ entrypoint; value } = transaction in
```

Comment:
What about record pattern here:
```ocaml
  let parse_transaction Tezos_bridge.Listen_transaction.{ entrypoint; value } =
    let value = Micheline.root value in
```
since we don't need the `transaction` but only each sub-field.

Reply:
I personally don't like it, the function header now has so much more noise, before I could read as `parse_transaction transaction` which is the function parse_transaction that accepts a transaction as a parameter, on your code I can see that it accept a record with `entrypoint` and `value`, but I don't know without looking at the type on what is the name that you give to this record.

Looking at the type is okay, but essentially my point is you removed a very useful name.